README.md

```diff
@@ -67,6 +67,42 @@ for your account at
 When generating a new personal access token, the scope only needs to
 encompass public_repo (or repo if you're accessing a private repo).
 
+### Test contexts
+
+For describing tests which we want to run we use __contexts__. Context has form:
+
+```
+os_image[/scenario][@bots#bots_pr][@owner/project/ref]
+```
+where items have the following meaning:
+- os_image: Name of the image on which tests should run (e.g. 'fedora-testing').
+- scenario: Name of specific test. This is unique for each separate project.
```

Comment:
Let's mention that this is passed verbatim to test/run in `$TEST_SCENARIO`. Also, it's not "unique" (projects can have many scenarios), it's "specific".